tools/render-test/options.cpp

```diff
@@ -223,6 +223,14 @@ static gfx::DeviceType _toRenderType(Slang::RenderApiType apiType)
             //
             // TODO: At some point we could warn/error and deprecate this option.
         }
+        else if (argValue == "-allow-glsl")
```

Comment:
In my latest change merged to top of tree, we may be able to use -lang glsl instead of this flag.